app/api/ticket_fees.py

```diff
@@ -1,15 +1,34 @@
 from flask_rest_jsonapi import ResourceDetail, ResourceList
-
+from sqlalchemy.orm.exc import NoResultFound
 from app import db
 from app.api.bootstrap import api
 from app.api.schema.ticket_fees import TicketFeesSchema
 from app.models.ticket_fee import TicketFees
+from app.api.helpers.exceptions import ConflictException
 
 
 class TicketFeeList(ResourceList):
     """
     List and create TicketFees
     """
+
+    def before_post(self, args, kwargs, data):
+        """
+        method to check for existing country currency combination
+        :param args:
+        :param kwargs:
+        :param data:
+        :return:
+        """
+        if (data['country'] and data['currency']):
+            try:
+                already_existing_combination = TicketFees.query.filter_by(country=data['country'],
+                                                                          currency=data['currency']).one()
```

Comment:
Just add a unique key on country and currency. No need for this check

Reply:
@iamareebjamal A unique key constraint would ensure that I don't get to save similar values of the same column.

I want to be able to save :`USA - USD` and `USA-INR` combination simultaneously. I don't think adding unique constraints would allow that

Reply:
You are simulating unique key constraint on TicketFee, if unique key won't work, this shouldn't work too

Reply:
You are ensuring that country and currency pair is unique and not repeated. That's the work on unique key